org-code-codebuilderapp/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Ignore Gradle project-specific cache directory
```

Comment:
should probably add .class files to the gitignore and remove them from this pr :)

Reply:
Oops! I didn't realize I was still on the same branch as this PR. I just kept pushing my work-in-progress.